components/brave_rewards/browser/rewards_service_impl.cc

```diff
@@ -1444,6 +1445,54 @@ void RewardsServiceImpl::SetConfirmationsIsReady(const bool is_ready) {
     ads_service->SetConfirmationsIsReady(is_ready);
 }
 
+void RewardsServiceImpl::GetAdsNotificationsHistory(
+    const uint64_t from_timestamp,
+    const uint64_t to_timestamp) {
+  if (!Connected()) {
+    return;
+  }
+
+  bat_ledger_->GetAdsNotificationsHistory(from_timestamp, to_timestamp,
+      base::BindOnce(&RewardsServiceImpl::OnGetAdsNotificationsHistoryMojoProxy,
+        AsWeakPtr()));
+}
+
+void RewardsServiceImpl::OnGetAdsNotificationsHistoryMojoProxy(
+    const std::string& transactions) {
+  std::unique_ptr<ledger::TransactionsInfo> info;
+  if (!transactions.empty()) {
+    info.reset(new ledger::TransactionsInfo());
+    info->FromJson(transactions);
+  }
+  OnGetAdsNotificationsHistory(std::move(info));
+}
+
+void RewardsServiceImpl::OnGetAdsNotificationsHistory(
+    std::unique_ptr<ledger::TransactionsInfo> transactionInfo) {
```

Comment:
`transactionInfo` should be either `transaction_info` or for consistency `info` to adhere to coding standards